### DISCUSSION TITLE
chrome确实不能用啊

### DISCUSSION THREAD

**Discussion by yaolongda** [ronggang/PT-Plugin-Plus#1386] [General]
![QQ截图20230403085328](https://user-images.githubusercontent.com/57882615/229389340-24f7e04f-db66-4ace-b10f-3adc9046ecb7.png)
![QQ截图20230403085346](https://user-images.githubusercontent.com/57882615/229389345-7c35c515-95a9-4aa1-b9b2-8886260fb7de.png)
我之前一直用的edge，就前几天edge变卡用回谷歌。昨天晚上插件还在用，今天就被谷歌删了，我还是解压安装的，真离谱。
没别的意思，就是跟大伙说一下，谷歌上就别用了，省得麻烦。
或者说只有我一个人遇到了这个问题QAQ

**Comment by mitlearn** [comment]
这边建议阅读安装方式 推荐crx https://github.com/pt-plugins/PT-Plugin-Plus/wiki#%E5%AE%89%E8%A3%85%E4%B8%8E%E6%9B%B4%E6%96%B0

**Comment by yaolongda** [comment]
@mitlearn 行，我用几天看看。
